app/models/boarding_point.rb

```diff
@@ -0,0 +1,10 @@
+class BoardingPoint < ApplicationRecord
+  include Resources
+  include WhitespaceValidation
+
+  has_many :boardings, dependent: :nullify
+  has_many :trips, through: :boardings
+
+  validates :name, presence: true, format: { with: VALID_LOCATION_REGEX }
+  validates :name, uniqueness: { case_sensitive: false }
```

Comment:
ditto!

Reply:
Apu, I had to break down this line to two different lines because `validates :name, presence: true, format: { with: VALID_LOCATION_REGEX }, uniqueness: { case_sensitive: false }` exceeds the maxLineLength of Rubocop.
In a previous review, Maliha Apu suggested me to use 110 maxLineLength for Rubocop.